media/inf/ks.inf

```diff
@@ -596,7 +585,6 @@ Pin.MultitrackRecorder="Multitrack Recorder"
 Pin.Synthesizer="Synthesizer"
 Pin.SWSynth="SW Synth"
 Pin.SWMidi="ReactOS GS Wavetable SW Synth"
-
```

Comment:
For example, those newlines were to separate the strings related to the different "modules" (Pin, AudFName, etc.).

Reply:
I am aware of that. Like I elaborated several times: It is much easier for translators if they do not have to replicate a specific set of newline-subsections a the correct place. Gets more important the larger the sections will get.
Makes review of future translation PRs easier if we know, that is formatted vertically correctly, if no newlines are in there at all. Keep in mind: The english source is not visible in a PR to double-check whether translators replicated it correctly or not. Splitting Localizable vs non-localizable should be the only place where we put a newline. That is what all of this is about. Intended.